spec/api/user_spec.rb

```diff
@@ -521,6 +521,14 @@ def build_structure_and_response(course_id, authors, build_initial_stats = true,
         expect(res["user_stats"]).to eq expected_result
       end
 
+      it "API doesn't fail if user has no activity" do
```

Comment:
Rename test to something like `"handle stats for user with no activity"`

Reply:
updated